framework/src/modules/http_api/controllers/node.js

```diff
@@ -251,22 +251,10 @@ NodeController.getStatus = async (context, next) => {
 			secondsSinceEpoch,
 			loaded,
 			syncing,
-			unconfirmedTransactions,
 			lastBlock,
 		} = await library.channel.invoke('chain:getNodeStatus');
 
-		// get confirmed count from cache or chain
-
-		const [confirmed, networkHeight] = await Promise.all([
-			_getConfirmedTransactionCount(),
```

Comment:
I think declaration of this function also can be removed

Reply:
done